### DISCUSSION TITLE
web.whatsapp.com QR code to link mobile app not recognised by whatsapp QR scanner, is there plans to support this without having to disable or tweak my setup?

### DISCUSSION THREAD

**Discussion by markusd1984** [darkreader/darkreader#13544] [General]
Whatsapp web app is probably very popular and several times got caught in the QR code not being recognised due to darkreader not optimised for it.

Can't remember where the right place is to report this, for a custom tweak to be added.

It looks like the QR scanner needs white background around a QR code image to work? lol

**Comment by Myshor** [comment]
There was some custom sitefix for that but they change code from time to time.

I will make fix to the web.whatsapp.com sitefix in a minute.

**Reply by Myshor** [reply]
Fix in https://github.com/darkreader/darkreader/pull/13561
